Source/WebKit/UIProcess/ios/forms/WKDatePickerPopoverController.mm

```diff
@@ -225,6 +261,14 @@ - (void)resetDatePicker
     [_delegate datePickerPopoverControllerDidReset:self];
 }
 
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/WKDatePickerPopoverControllerAdditions.mm>)
+#import <WebKitAdditions/WKDatePickerPopoverControllerAdditions.mm>
+#else
+- (void)viewSafeAreaInsetsDidChange
+{
+}
+#endif
```

Comment:
Can be simplified.
```suggestion
#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/WKDatePickerPopoverControllerAdditions.mm>)
#import <WebKitAdditions/WKDatePickerPopoverControllerAdditions.mm>
#endif
```

The default implementation of `viewSafeAreaInsetsDidChange` will be used.